spinn_utilities/log.py

```diff
@@ -174,6 +176,21 @@ def set_kill_level(cls, level=None):
         else:
             cls.__kill_level = level
 
+    @classmethod
+    def set_report_File(cls, report_file):
+        """
+
+        :param report_file:
+        :param write_normal:
+        :return:
+        """
+        cls.__report_file = report_file
+        level = logging.getLevelName(cls.__repeat_at_end)
+        with open(report_file, "a") as report_file:
+            report_file.write(
+                "This is a record of all logged messages at level {} or "
+                "above\n".format(level))
```

Comment:
Do we have easy access anywhere here to add either a job ID or a timestamp?

Reply:
There is no need to do that here due to the file Fron end common picks

There will be a separate report file for each run in the report director such as

reports/2020-12-15-07-44-21-617755/run_1/